integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java

```diff
@@ -194,6 +197,20 @@ private void createBindUser(TestContext testContext, com.sequenceiq.freeipa.api.
         }
     }
 
+    private void generateHostKeyTab(com.sequenceiq.freeipa.api.client.FreeIpaClient ipaClient, String environmentCrn) {
+        try {
+            HostKeytabRequest hostKeytabRequest = new HostKeytabRequest();
+            hostKeytabRequest.setEnvironmentCrn(environmentCrn);
+            hostKeytabRequest.setServerHostName("test.local");
+            hostKeytabRequest.setRoleRequest(new RoleRequest());
```

Comment:
have you tested this? does this work? I think if `RoleRequest` is not `null` then `roleName` must have a value

Reply:
please drop this line, it's not necessary, otherwise the pr is ok

Reply:
Done.